test/common.py

```diff
@@ -469,3 +469,149 @@ def set_rledata(self,data):
             temp = temp >> 4
             if(temp):
                 print("%s,%s,%s"%(amplitude,runlength,size))
+
+class Huffman:
+    def __init__ (self):
+        """
+        This class stores the value of the matrixes used to test the Huffman module.
+        These matrixes were taken from [`huffman_test_inputs.py` in cfelton's test_jpeg code]
+        (https://github.com/cfelton/test_jpeg/blob/master/test/huff_test_inputs.py).
+        The matrix is an example of what the quantization module might produce.
+        """
```

Comment:
Indenting...
```python
    self.vli_test_y = [
        10, 1, 2, 4, 8, 16, 32, 65,
        128, 256, 1000, 1, 2, 4, 8, 16,
        32, 65, 128, 256, 1000, 1, 2, 4,
        8, 16, 32, 65, 128, 256, 1, 3,
        3, 3, 3, 3, 3, 3, 3, 3,
        3, 3, 3, 3, 3, 3, 3, 3,
        3, 3, 3, 3, 3, 3, 3, 3,
        2, 2, 2, 2, 3, 3, 3, 2,
    ]
```

Reply:
Done

Reply:
This doesn't look to be done?

Reply:
@mithro, Do you want something else because I have already indented the values written in the matrix in proper order.
Kindly update if anything is missing.

Reply:
See my example in the first comment here -> https://github.com/timvideos/litejpeg/pull/3/files#r137180177

Reply:
This comment is redirecting to the same page, doesn't able to understand.